### PR TITLE
CASMHMS-5103 Servertech power control

### PR DESCRIPTION
**Pull request by mjendrysik-hpe**
## Summary and Scope

Created a python tool that simplifies controlling power on Server Tech iPDU outlets. Individual outlets and outlet groupings can be controlled.

## Issues and Related PRs

* Resolves [CASMTRIAGE-5103](https://jira-pro.its.hpecorp.net:8443/browse/CASMTRIAGE-5103)

## Testing

Tested on:

  * `gamora`

Test description:

Did testing of individual command lines by hand. All combinations were tested including using a json file as input and overriding settings made in the json.

## Risks and Mitigations

No known risks

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,2 @@
-# Example Github CODEOWNERS file
-# See https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners for more information.
-# This is a comment.
-# Each line is a file pattern followed by one or more owners.
-#
-# In general, CODEOWNERS should be team in the Cray-HPE organization, not individuals.
-#
+# Hardware Management Team Members and Administrators
+*  @Cray-HPE/hardware-management @Cray-HPE/hardware-management-admin
```
Comment (rsjostrand-hpe) [inline .github/CODEOWNERS:2]: @mjendrysik-hpe I assume you are the admin of this repo? Can you add the  `@Cray-HPE/hardware-management` group with maintain permissions, and `@Cray-HPE/hardware-management-admin` with admin permissions?

File: README.md [hunk 2]
```diff
@@ -1,66 +1,255 @@
...
+
+optional arguments:
+  -h, --help         show this help message and exit
+  --on               turn selected outlets/groups On
+  --off              turn selected outlets/groups Off
+  --reboot           reboot selected outlets/groups
+  --status           get status of outlet or group
+  --pdus PDUS        iPDU controller hostnames, IPv4 addrs, and/or IPv6 addrs
+  --outlets OUTLETS  target outlets
+  --groups GROUPS    target groups
```
Comment (rsjostrand-hpe) [inline README.md:52]: Are these groups the same thing as outlet groups configured on the PDU, or are they a concept of this tool? If they are configured on the PDUs what creates these groups?

Comment (mjendrysik-hpe) [reply README.md:52]: They are the groups on the iPDU themselves. Site personnel are working on creating the groups. In the end, this tool should handle all of it IMO.

File: .github/CODEOWNERS (hunk 1 above)
Comment (mjendrysik-hpe) [reply .github/CODEOWNERS:2]: Done

**Review by shunr-hpe — APPROVED** [review]

**Pull request merged**